docs/samples/Microsoft.ML.Samples.FSharp/Microsoft.ML.FSharp.Samples.fsproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="Dynamic\Transforms\Text\ApplyCustomWordEmbedding.fs" />
+    <Compile Include="Program.fs" />
```

Comment:
So, is this necessary? With `.csproj` you don't need to enumerate files, is it actualyl necesary for FSharp?